.github/workflows/benchmark.yml

```
@@ -0,0 +1,127 @@
+  IMAGE: appwrite-dev
+  CACHE_KEY: appwrite-dev-${{ github.event.pull_request.head.sha }}
+
+on: [ pull_request ]
+
+jobs:
+  setup:
+    name: Setup & Build Appwrite Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)